accepted/0000-hub_retail.md

```diff
@@ -0,0 +1,310 @@
+- Feature Name: Hub Retail Extension
+- Start Date: 2020-11-30
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC describes an extension of retail features for Hub environment.
+
+# Motivation
+[motivation]: #motivation
+
+Reasons for use of Hub with Retail environment
+----------------------------------------------
+
+- scalability
+  - support large environments with the number of terminal exceeding limits of one SUMA server
+
+- disconnected operation
+  - Peripheral server must stay fully functional when the Hub connection is down
+
+See [Hub requirements](https://github.com/SUSE/spacewalk/wiki/Hub-requirements) for details.
+
+Possible configurations
+-----------------------
+
+1. Standard
+```
+    Terminal -> Branch Server -> SUMA Server
+```
+  This is the currently supported configuration.
+  It is suitable for one large department store.
```

Comment:
> It is suitable for one large department store.

Really? Maybe I have different definition of department store, but we have users operating multiple department stores. This is a nitpick, but IMO this makes disservice to the solution.

Reply:
As I wrote on the next line, larger deployments have the problems that we are trying to solve by this RFC.